### PR TITLE
Add needed steps in README

### LINKED ISSUE #1: Is README missing a step in the installation?
This looks great, thanks for making a Matomo DDEV add-on!

I followed the steps in the README, but can't get past `3. Database Setup` where I get a `Could not connect to the database server: Database access denied ` message. Also, it looks like you need to restart DDEV to register the add-on, so I'll start a PR, to clarify this, and maybe other things in the README.

### PR DESCRIPTION
**Pull request by gitressa**
- Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 # ddev-matomo <!-- omit in toc -->
 
 
-## What is ddev-matomo?
+## What is DDEV Matomo?
```
Comment (rfay) [inline README.md]: I prefer the old way.

File: README.md [hunk 2]
```diff
@@ -14,4 +14,5 @@ more than powerful analytics.
 1. Install the plugin with `ddev get valthebald/ddev-matomo`
 2. Review your database settings in .ddev/docker-compose.matomo.yaml
 3. Create matomo database if needed with `echo "SHOW TABLES;" | ddev import-db --database=${MATOMO_DATABASE_DBNAME}`'
-4. Access https://matomo.\<projectname\>.ddev.site in your browser and follow the installation steps.
+4. Restart DDEV to install DDEV Matomo with `ddev restart`
```
Comment (rfay) [inline README.md]: I'm afraid I don't like this here, don't think it adds value.

File: README.md (hunk 1 above)
Comment (gitressa) [reply README.md]: But this is from the style guide:

> 12. Update the `README.md` header in Title Case format, for example, use `# DDEV Redis`, not `# ddev-redis`.

https://github.com/ddev/ddev-addon-template?tab=readme-ov-file#getting-started

File: README.md (hunk 2 above)
Comment (gitressa) [reply README.md]: But if you proceed to the next step, will Matomo be ready without a `ddev restart`?

File: README.md (hunk 1 above)
Comment (gitressa) [reply README.md]: See also https://github.com/ddev/ddev/issues/5641.

Comment (gitressa) [reply README.md]: The style guide got updated to use ddev-matomo.

File: README.md (hunk 2 above)
Comment (gitressa) [reply README.md]: Does it work for you?

Comment (valthebald) [reply README.md]: @gitressa Indeed, ddev restart is required!

**Comment by valthebald** [comment]
@gitressa merged to main, thank you!

**Comment by gitressa** [comment]
Great, thanks @valthebald!

**Pull request merged**